### DISCUSSION TITLE
LanguageTool spell checking or synonyms using Wordnet

### DISCUSSION THREAD

**Discussion by sdanfa** [remirror/remirror#1908] [Ideas]
Hi,
I need a functionality to do better spell checking using LanguageTool and a drop down that gives synonyms using Wordnet.
I saw there's a prosemirror implementation that uses another tool Typerighter which calls to LanguageTool for spell checking:  https://github.com/guardian/prosemirror-typerighter , I can't figure out how to make use of it with remirror. Could someone help?

There's also https://wordnet.princeton.edu/ which is a lexical database for synonyms, is there a plugin for it?

Thanks!

**Comment by ocavue** [comment]
> is there a plugin for it?

As far as I know, no. But we welcome contribution.
